README.md

```diff
@@ -175,8 +175,8 @@ var storage = multer.diskStorage({
     cb(null, '/tmp/my-uploads')
   },
   filename: function (req, file, cb) {
-    const uniqueHash = Date.now() + '-' + Math.random().split('.')[1]
-    cb(null, file.fieldname + '-' + uniqueHash)
+    const uniqueSuffix = Date.now() + '_' + (Math.random().toString().split('.')[1])
```

Comment:
What do you think about `Math.round(Math.random() * 1E9)` instead of `(Math.random().toString().split('.')[1])`, I feel that it's a bit easier to understand?

Reply:
If that works, I'm ok with it.